modules/ROOT/pages/managing-instances/instance-details.adoc

```diff
@@ -54,6 +54,28 @@ A snapshot is a copy of the data in an instance at a specific point in time.
 Neo4j regularly takes snapshots of your instance, and you can request a snapshot to be taken on demand.
 These snapshots can be used to restore data to a different Neo4j instance.
 
+== Vector optimization
+
+label:AuraDB-Professional[]
+label:AuraDB-Business-Critical[]
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+Vector optimization reserves memory for vector indexes, enhancing performance for vector-based operations.
+It is available for AuraDB instances with more than 4GB of memory, and is available across all supported cloud providers and regions.
+
+This configuration may impact memory use and therefore may affect performance for other queries.
+
+You can enable vector optimization during instance creation when selecting *Instance details* with a toggle under *Additional settings*.
```

Comment:
```suggestion
You can enable vector optimization during instance creation using the toggle in *Additional settings*, in the *Instance details*.
```

Reply:
Thanks for the suggestion - I considered both phrases, and the original phrasing seems a bit clearer as it guides users through a sequential flow, directing them to Instance details before mentioning Additional settings. Let me know if you have any other thoughts!

Reply:
Update: waiting for the final UX for the setting before finalizing this part